examples/README.md

```diff
@@ -0,0 +1,2 @@
+# Examples
+This directory contains code segments showing how to utilize this Python package.
```

Comment:
This will all live in either the readme, if it is SDK specific.  If it is showing full flows, there is a different repo for samples.

Reply:
Noted, I'll remove it